.github/workflows/system-tests.yaml

```
@@ -0,0 +1,15 @@
+name: System tests
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  run-system-tests:
+    if: ${{ github.event.label.name == 'trigger-system-tests' }}
+    uses: precice/tutorials/.github/workflows/run_testsuite_workflow.yml@develop
+    with:
+      suites: release_test
+      build_args: PRECICE_REF:${{ github.event.pull_request.head.sha }},OPENFOAM_EXECUTABLE:openfoam2312,OPENFOAM_ADAPTER_REF:master,TUTORIALS_REF:develop,PLATFORM:ubuntu_2204,PYTHON_BINDINGS_REF:master,FENICS_ADAPTER_REF:master,CALULIX_VERSION:2.20,CALULIX_ADAPTER_REF:master,SU2_VERSION:7.5.1,SU2_ADAPTER_REF:master
```

The `CALULIX_VERSION` needs to be fixed here as well. From the `stdout.log` of the perpendicular flap with OpenFOAM and CalculiX:

```
#23 [solid-calculix calculix_adapter 4/6] RUN wget http://www.dhondt.de/ccx_${CALULIX_VERSION}.src.tar.bz2 &&     tar xvjf ccx_${CALULIX_VERSION}.src.tar.bz2 &&     rm -fv ccx_${CALULIX_VERSION}.src.tar.bz2
#23 0.214 --2024-07-18 09:41:37--  http://www.dhondt.de/ccx_.src.tar.bz2
#23 0.214 Resolving www.dhondt.de (www.dhondt.de)... 81.169.145.94, 2a01:238:20a:202:1094::
#23 0.249 Connecting to www.dhondt.de (www.dhondt.de)|81.169.145.94|:80... connected.
#23 0.253 HTTP request sent, awaiting response... 404 Not Found
#23 0.258 2024-07-18 09:41:37 ERROR 404: Not Found.
#23 0.258 
#23 ERROR: process "/bin/bash -c wget http://www.dhondt.de/ccx_${CALULIX_VERSION}.src.tar.bz2 &&     tar xvjf ccx_${CALULIX_VERSION}.src.tar.bz2 &&     rm -fv ccx_${CALULIX_VERSION}.src.tar.bz2" did not complete successfully: exit code: 8
```

---

```suggestion
      build_args: PRECICE_REF:${{ github.event.pull_request.head.sha }},OPENFOAM_EXECUTABLE:openfoam2312,OPENFOAM_ADAPTER_REF:master,TUTORIALS_REF:develop,PLATFORM:ubuntu_2204,PYTHON_BINDINGS_REF:master,FENICS_ADAPTER_REF:master,CALCULIX_VERSION:2.20,CALCULIX_ADAPTER_REF:master,SU2_VERSION:7.5.1,SU2_ADAPTER_REF:master
```

It's a typo anyway, let's see where else we need to update what.